### ISSUE TITLE
Claims Angular 2 - 6 compatible, but only Angular 6 supported.

### ISSUE THREAD

**Issue by shaneray** [Bartosz-D3V/ng-payment-card#190]
When attempting to use this with Angular 4 I am getting error, `i0.defineInjectable is not a function`.  It appears `defineInjectable` is an Angular 6 only thing.

**Comment by Bartosz-D3V** [comment]
Hi @shaneray 
Thank you for raising the issue.
I do not use ```defineInjectable``` 'manually' so it looks like one of the dependencies should be downgraded.
I will take a look.

Many thanks.

**Comment by shaneray** [comment]
@Bartosz-D3V Thanks for the quick response and thank you for open sourcing such a nice library.  I am in a time crunch so will have to look for others or implement my own, but wanted to provide a bit more detail to help you get to the bottom of this.  I am somewhat new to Angular 2+ and noticed there was a piece missing from the install details that may help others as well.. After doing the `import { NgPaymentCardModule } from 'ng-payment-card';` in the app.shared.module I also had to add it to the `imports:` on the @NgModule.

*Example*
```
@NgModule({
  imports: [..., NgPaymentCardModule ],
  declarations: [...],
})
```

*Error Details*
NodeInvocationException: Prerendering failed because of error: TypeError: i0.defineInjectable is not a function at CardValidator.numbersOnly.ccNum

*Addition Info*
Using Angular: 4.3.0, with pre-rendering enabled.  Also tried without pre-rendering and received same error.

**Comment by Bartosz-D3V** [comment]
Hi @shaneray 
Once again, thank you so much for raising this bug.
I have fixed it and it should be resolved as part of version 0.0.6.

Please do not hesitate to contact if you will have any other questions/issues.
